csrankings-x.csv

```diff
@@ -444,6 +444,7 @@ Xu Liu 0001,North Carolina State University,https://xl10.github.io,qhLT67EAAAAJ
 Xu Liu 0006,Xidian University,https://web.xidian.edu.cn/xuliu/,_09bkMgAAAAJ
 Xu Sun 0001,Peking University,https://xusun.org,NOSCHOLARPAGE
 Xu Tang,Xidian University,https://sites.google.com/site/xutanghomepage/,lmskeBMAAAAJ
+Xu Wang 0006,Shenzhen University,https://openreview.net/profile?id=~Xu_Wang17,XNDHhIEAAAAJ

```

Comment:
Replace openreview with actual home page

Reply:
I post the official website homepage link and screenshot from College of Computer Science and Software Engineering in Shenzhen University. Please check if this homepage is eligible? Thank you so much.
[https://csse.szu.edu.cn/en/pages/user/index?id=556](url)
![Uploading Xu Wang.jpg…]()